libevmasm/Assembly.cpp

```diff
@@ -353,7 +353,7 @@ void Assembly::injectStart(AssemblyItem const& _i)
 	m_items.insert(m_items.begin(), _i);
 }
 
-Assembly& Assembly::optimise(bool _enable, bool _isCreation, size_t _runs)
+Assembly& Assembly::optimise(bool _enable, EVMVersion _evmVersion, bool _isCreation, size_t _runs)
```

Comment:
Woohoo, I should be using this for the bitwise shifting optimiser rules in #3580.

Reply:
Yes, it was a really painful process. This setting is needed literally everywhere...